jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+(a.k.a. <<Author>>) should first attempt to ascertain whether the idea is JEP-able.
+Posting to the jenkinsci-dev@googlegroups.com mailing list is the best way to
+go about this.
+
+[[discussion]]
+==== Discuss the idea with the community
+
+Vetting an idea publicly before going as far as writing a JEP is meant
+to save the potential author time. Many ideas have been brought
+forward for changing Jenkins that have been rejected for various
+reasons. Asking the Jenkins community first if an idea is original
+helps prevent too much time being spent on something that is
+guaranteed to be rejected based on prior discussions (searching
+the internet does not always do the trick). It also helps to make sure
+the idea is applicable to the entire community and not just the author. Just
+because an idea sounds good to the author does not mean it will work for most
+people in most areas where Jenkins is used.
+
+Once the champion has asked the Jenkins community as to whether an idea has any
+chance of acceptance, a draft JEP should be presented to jenkinsci-dev@. This
+gives the author a chance to flesh out the draft JEP to make sure it is
+properly formatted, of high quality, and to address initial concerns about the
+proposal.
+
+[[submission]]
+==== Creating a JEP Submission
+
+Following a discussion on jenkinsci-dev@,
+the proposal should be turned into as a JEP submission and submitted
+via a GitHub pull request to this repository footnoteref:[repo].
+The submission must be written in JEP style as described below,
+otherwise, it will fail review immediately
+(although minor errors may be corrected by the editors).
+
+To submit a JEP for approval as Draft, the JEP author should:
+
+. Fork the JEP repository footnoteref:[repo].
+. Create a new branch called `jep-submission` their fork
+  (`git checkout -b jep-submission`).
+. Copy the folder `jep-template/0000` to `jep/0000`.
+. Modify the template JEP in `jep/0000`
+  per the instructions in this JEP (which are also outlined in the template).
+. Commit and push the changes to their fork
+  and submit a pull request targeting the `jenkinsci/master` branch.
+
+A JEP editor will check the submission for conformance with
+JEP structure and formatting guidelines.
+Editors may make minor changes to make the submission meets
+the requirements for approval as a Draft JEP.
+If a JEP requires major changes, editors will add specific feedback
+and send the submission back to the author for revision.
+
+IMPORTANT: All submissions must go through pull request,
+even those by editors or contributors with "git push" privileges
+for the JEP repository footnoteref:[repo].
+
+The JEP editors will not unreasonably deny a JEP.
+Reasons for denying JEP "Draft" status include:
+
+* duplication of effort
+* being technically unsound
+* not providing enough information in the
+  "Motivation" or "Backwards Compatibility" sections
+* not in keeping with the Jenkins philosophy.
```

Comment:
It would be great to describe it somewhere. I am pretty sure everyone has different understanding of Jenkins philosophy